VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -108,6 +117,8 @@ describe('ParticipantsPanelComponent', () => {
     });
 
     it('should get participant sorted list, the judge is first, then panel members and finally observers are the last one', fakeAsync(() => {
+        const mappedParticipants = mapper.mapFromParticipantUserResponseArray(participants);
+        participantPanelModelMapperSpy.mapFromParticipantUserResponseArray.and.returnValue(mappedParticipants);
```

Comment:
Can't participantPanelModelMapperSpy.mapFromParticipantUserResponseArray.and.callThrough be used instead?

Reply:
Yeah, that's true. I guess I was in between two methodologies here when rewriting these tests.